lib/model.js

```diff
@@ -314,6 +314,25 @@ Model._getAccessTypeForMethod = function(method) {
 
   var ACL = Model._ACL();
 
+  // Check the explicit setting of accessType
+  if (method.accessType) {
+    assert(method.accessType === ACL.READ ||
+      method.accessType === ACL.WRITE ||
+      method.accessType === ACL.EXECUTE, 'invalid accessType ' +
+      method.accessType +
+      '. It must be "READ", "WRITE", or "EXECUTE"');
+    return method.accessType;
+  }
+
+  // Default GET requests to READ
+  var verb = method.http && method.http.verb;
+  if (typeof verb === 'string') {
+    verb = verb.toUpperCase();
+  }
+  if (verb === 'GET' || verb === 'HEAD') {
+    return ACL.READ;
```

Comment:
Maybe we should allow the user to specify this via `config.json`. You could just check `app.get('access').defaultAccessTypes[verb]`.